src/tests/suite/dfs_unit_test.c

```diff
@@ -3053,6 +3053,107 @@ dfs_test_fix_chunk_size(void **state)
 	D_FREE(buf);
 }
 
+static void
+dfs_test_oflags_write_file(char *name)
```

Comment:
you can use dfs_test_file_gen() instead of adding this?

Reply:
You are right. I am going to use dfs_test_file_gen() instead. Thank you!